client/webserver/site/src/js/register.js

```diff
@@ -21,7 +21,7 @@ export default class RegistrationPage extends BasePage {
       'newWalletForm',
       // Form 3: Unlock Decred wallet
       'unlockWalletForm',
-      // Form 4: PreRegister DEX
+      // Form 4: Register DEX
```

Comment:
Uhm... this should be `Check DEX Fees`, the actual registration is performed by `Form 5` below this.
I see it was previously `Form 4: DEX address`, but that was erroneously changed to `PreRegister DEX` in #326.

Reply:
Ah, so it was.  Good catch.  I revised it to "Configure DEX server".  Anyway, it's just a comment.